src/js/utils/helpers.js

```diff
@@ -296,7 +296,7 @@ define([
             if(duration <= MIN_DVR_DURATION) {
                 return 'DVR';
             }
-            if (duration < 0 || duration === Infinity) {
+            if (duration === null || duration < 0 || duration === Infinity) {
```

Comment:
Can you add a comment here, for each possible duration and why that means live?

// Infinity is Safari, <0 is DVR, null is HLS?